dags/kpi_dashboard.py

```diff
@@ -30,18 +33,65 @@
         email=['telemetry-alerts@mozilla.com', 'jklukas@mozilla.com']
     )
 
-    smoot_usage_new_profiles_v2 = bigquery_etl_query(
-        task_id='smoot_usage_new_profiles_v2',
+    simpleprophet_forecasts_mobile = simpleprophet_forecast(
+        task_id="simpleprophet_forecasts_mobile",
+        datasource="mobile",
         project_id='moz-fx-data-shared-prod',
-        destination_table='smoot_usage_new_profiles_v2',
         dataset_id='telemetry_derived',
+        table_id='simpleprophet_forecasts_mobile_v1',
+        owner="jklukas@mozilla.com",
+        email=["telemetry-alerts@mozilla.com", "jklukas@mozilla.com"],
     )
 
-    smoot_usage_new_profiles_compressed_v2 = bigquery_etl_query(
-        task_id='smoot_usage_new_profiles_compressed_v2',
+    wait_for_firefox_nondesktop_exact_mau28 = ExternalTaskSensor(
+        task_id="wait_for_firefox_nondesktop_exact_mau28",
+        external_dag_id="bqetl_nondesktop",
+        external_task_id="telemetry__firefox_nondesktop_exact_mau28_raw__v1",
+        check_existence=True,
+        mode="reschedule",
+        dag=dag,
+    )
+
+    simpleprophet_forecasts_mobile.set_upstream(wait_for_firefox_nondesktop_exact_mau28)
+
+    simpleprophet_forecasts_desktop = simpleprophet_forecast(
+        task_id="simpleprophet_forecasts_desktop",
+        datasource="desktop",
         project_id='moz-fx-data-shared-prod',
-        destination_table='smoot_usage_new_profiles_compressed_v2',
         dataset_id='telemetry_derived',
+        table_id='simpleprophet_forecasts_desktop_v1',
+        owner="jklukas@mozilla.com",
+        email=["telemetry-alerts@mozilla.com", "jklukas@mozilla.com"],
+        dag=dag)
+
+    wait_for_exact_mau_by_dimensions = ExternalTaskSensor(
+        task_id="wait_for_exact_mau_by_dimensions",
+        external_dag_id="main_summary",
+        external_task_id="exact_mau_by_dimensions",
+        check_existence=True,
+        mode="reschedule",
```

Comment:
We might want to use this mode for `ExternalTaskSensor` in general. 
_"When set to reschedule the sensor task frees the worker slot when the criteria is not yet met and it’s rescheduled at a later time."_ (see https://airflow.apache.org/docs/stable/_api/airflow/sensors/base_sensor_operator/index.html)